tests/boulder-integration.sh

```diff
@@ -11,6 +11,8 @@
 . ./tests/integration/_common.sh
 export PATH="/usr/sbin:$PATH"  # /usr/sbin/nginx
 
+export GOPATH="${GOPATH:-/tmp/go}"
```

Comment:
From `man dash`:

" ${parameter:-word}    Use Default Values.  If parameter is unset or null, the expansion of word is substituted; otherwise, the value of parameter is substituted."

In short: we default to `/tmp/go` if `GOPATH` is not already set.